airflow/utils/helpers.py

```diff
@@ -202,3 +203,11 @@ def cross_downstream(*args, **kwargs):
         stacklevel=2,
     )
     return import_string('airflow.models.baseoperator.cross_downstream')(*args, **kwargs)
+
+
+def is_k8s_or_k8scelery_executor() -> bool:
+    """Determines if the executor utilizes k8s."""
+    conf_executor = conf.get('core', 'EXECUTOR')
+    if conf_executor in ('KubernetesExecutor', 'CeleryKubernetesExecutor'):
```

Comment:
Let use ExecutorLoader instead of hardcoding names of executors

Reply:
@turbaszek I had to undo this as it was creating a cyclical import